GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -2053,6 +2053,16 @@ private string GetRefUnambiguousName(IGitRef gitRef)
                 : gitRef.Name;
         }
 
+        private void RebaseOnToolStripMenuItem_DropDownOpening(object sender, EventArgs e)
+        {
+            IReadOnlyList<GitRevision> selectedRevisions = GetSelectedRevisions();
+            rebaseToolStripMenuItem.Enabled
+                = rebaseInteractivelyToolStripMenuItem.Enabled
+                = _rebaseOnTopOf is not null && selectedRevisions.Count == 1;
+            rebaseWithAdvOptionsToolStripMenuItem.Enabled = _rebaseOnTopOf is not null && (selectedRevisions.Count == 1
+                || (selectedRevisions.Count == 2 && selectedRevisions.All(r => !r.IsArtificial)));
```

Comment:
```suggestion
            rebaseWithAdvOptionsToolStripMenuItem.Enabled = _rebaseOnTopOf is not null
                && (selectedRevisions.Count == 1
                    || (selectedRevisions.Count == 2 && selectedRevisions.All(r => !r.IsArtificial)));
```

Reply:
@mstv I have applied your suggestion....
Are wee good to go now?

Side note: I need to have my 2 PRs merged before tomorrow otherwise they will have to wait 1 week (AFKB)

Reply:
I believe this should be merged now

> Side note: I need to have my 2 PRs merged before tomorrow otherwise they will have to wait 1 week (AFKB)

or someone else merge

Reply:
> or someone else merge

Indeed but I was more thinking about if it needs some changes on my side....